posts/2023-09-01-committing-lockfiles.md

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
> posts/2023-09-01-committing-lockfiles.md

Currently I named this for next week because I didn't know how long discussions around this would go.  I can update it to whatever day we feel we are ready to merge this.

Reply:
I've updated the post to be for tomorrow for after the cargo ream meeting